tools/browser_compat_data_update.py

```diff
@@ -0,0 +1,113 @@
+import json
```

Comment:
`#!/usr/bin/env python3`

+ copyright 

+ docstring explaining what this script does.